pkg/refactor/mirror/callbacks.go

```diff
@@ -11,57 +10,10 @@ import (
 
 type Callbacks interface {
 	// ConfigureWorkflowFunc is a hookup function the gets called for every new or updated workflow file.
+	// TODO: alan, can we remove/replace this in favor for pubsub?
```

Comment:
This question is actually better suited to @sir-hassan right now, since he may have done something similar as part of his v2 filesystem API.